app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -155,7 +155,9 @@ interface Pixel {
         FEEDBACK_NEGATIVE_SUBMISSION("mfbs_%s_%s_%s"),
 
         AUTOCOMPLETE_BOOKMARK_SELECTION("m_aut_s_b"),
-        AUTOCOMPLETE_SEARCH_SELECTION("m_aut_s_s")
+        AUTOCOMPLETE_SEARCH_SELECTION("m_aut_s_s"),
+
+        CHANGE_APP_ICON_OPENED("mic"),
```

Comment:
We should be using `m_` prefix for all new pixels unless there's a good reason not to

Reply:
Changed it to `m_ic` 👍